lang/en.json

```diff
@@ -51,7 +51,6 @@
     "UI.REPAIRS": "Repairs",
     "UI.TRUNK": "Trunk",
     "UI.VEHICLEARMOR": "Vehicle Armor",
-    "UI.VEHICLEFULL": "The vehicle is full.",
```

Comment:
Removed this and the error condition because I think it is better to let the payers/gm manage the limitation. 

I guess there could be a situation where they squeeze some in the vehicle or in the trunk... so I don't think the system should enforce it.